resources/views/home/landing.blade.php

```diff
@@ -0,0 +1,203 @@
+{{--
+    Copyright 2015 ppy Pty. Ltd.
+
+    This file is part of osu!web. osu!web is distributed with the hope of
+    attracting more community contributions to the core ecosystem of osu!.
+
+    osu!web is free software: you can redistribute it and/or modify
+    it under the terms of the Affero GNU General Public License version 3
+    as published by the Free Software Foundation.
+
+    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+    See the GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+    --}}
+
+    @extends("master", [
+    'title' => 'osu!',
+    'blank' => 'true',
+    'body_additional_classes' => 'osu-layout--body-landing'
+    ])
+
+    @section("content")
+    <nav class="osu-layout__row osu-layout__row--landing">
+        <!-- Mobile Navigation -->
+        @include('objects.smartphone-header', ['navLinks' => landing_nav_links(), 'subLinks' => false])
```

Comment:
apart of the single comment, nothing else actually uses the term "smartphone". It's "mobile".